addons/towing/functions/fnc_towStateMachinePFH.sqf

```diff
@@ -140,7 +140,7 @@ switch (_state) do {
         [_unit, _ropeClass, true] call CBA_fnc_addItem;
         _args set [0, TOW_STATE_CLEANUP];
         GVAR(cancel) = false;
-
+        [QGVAR(ropeDeployedCanceled), [_unit, _ropeClass]] call CBA_fnc_localEvent;
```

Comment:
~~ropeDeployedCanceled~~
ropeDeployCanceled 

better name?

Reply:
Also Cancelled or Canceled ? xD

Reply:
Ahhhh. It should be cancelled